sccm/core/plan-design/configs/support-for-windows-10.md

```diff
@@ -79,3 +79,6 @@ The following table lists the versions of the Windows 10 ADK that you can use wi
 
  > [!Note]  
  > Configuration Manager only supports x86 and amd64 components of the Windows 10 ADK. It doesn't currently support ARM or ARM64 components. 
+
+# ADK version support for Server OS deployment: 
+Treat Server 2016 like Windows 10 LTSB 2016 as both of them are 10.14393.x
```

Comment:
Please tweak the wording on this, something like, 
`Windows Server builds have the same ADK requirement as the associated Windows 10 version. For example, Windows Server 2016 is the same build version as Windows 10 LTSB 2016.`
I think something simple like that is sufficient. What do you think?

Reply:
I agree with that thanks